docs/mobile-apps/automated-testing/appium/quickstart.md

```diff
@@ -90,4 +90,10 @@ cd quickstart-appium-java
 mvn clean test
 ```
 
+The terminal command ```mvn clean test``` uses a containing test classes configuration valued inside `<testngXmlFile>` attribute of `<properties>` attributes from the `pom.xml` file and the default configuration in the [Appium Demo Project](https://github.com/saucelabs-training/quickstart-appium-java) is the emulator (Android) configuration `appium_android_emu_web_test.xml`. 
+
+That results in executing tests inside of the class `src/test/java/tests/EmuSim/AppiumAndroidEmuWebTest` when `mvn clean test` is run. 
+
+Change the value of the `<testngXmlFile>` attribute to `appium_ios_rdc_app_test.xml` to use the real device (iOS) configuration with tests located inside the class `src/test/java/tests/RDC/AppiumIosRdcAppTest` and again run `mvn clean test`.
+
```

Comment:
```suggestion
```
I suggest we update the terminal command above for a real iOS device, and remove this information from the Quickstart since it's included in the [README.md file](https://github.com/saucelabs-training/quickstart-appium-java#readme) for the linked quickstart-appium-java demo repo.